docs/DeveloperGuide.md

```diff
@@ -76,7 +82,7 @@ Our strategy is to ensure all aspects of the generator are tested through some f
 
 * If a change is made to existing class implementations and a test does not exist, then a test should be added
 
-[JUnit (Jupiter)](https://junit.org/junit5/docs/current/user-guide/) is used for unit and integration tests. An outline of how unit tests should be written within DataHelix can be found [here](./developer/JUnitCookbook.md).
+[JUnit (Jupiter)](https://junit.org/junit5/docs/current/user-guide/) is used for unit and integration tests. This document shows [an outline of how unit tests should be written within DataHelix](./developer/JUnitCookbook.md).
```

Comment:
To me the "This Document" should be in the link

Reply:
Link text updated slightly